hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOMRatisSnapshots.java

```diff
@@ -198,12 +198,12 @@ public void shutdown() {
     }
   }
 
-  @ParameterizedTest
-  @ValueSource(ints = {100})
   // tried up to 1000 snapshots and this test works, but some of the
   //  timeouts have to be increased.
-  @Unhealthy("HDDS-10059")
-  void testInstallSnapshot(int numSnapshotsToCreate, @TempDir Path tempDir) throws Exception {
+  private static int numSnapshotsToCreate = 100;
```

Comment:
nit: can be final

```suggestion
  private static final int NUM_SNAPSHOTS_TO_CREATE = 100;
```

(needs corresponding change in `testInstallSnapshot`)